templates/case-studies/contact-us.html

```diff
@@ -0,0 +1,215 @@
+                <input required
+                       id="email"
+                       name="email"
+                       maxlength="255"
+                       type="email"
+                       pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone">Mobile/cell phone number:</label>
+                <input id="phone" name="phone" maxlength="255" type="tel" />
+              </li>
+            </ul>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <label class="p-checkbox">
+                  <input class="p-checkbox__input"
+                         value="yes"
+                         aria-labelledby="canonicalUpdatesOptIn"
+                         name="canonicalUpdatesOptIn"
+                         type="checkbox" />
+                  <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+                </label>
+              </li>
+              <li class="p-list__item u-sv3">
+                By submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.
+              </li>
+              {# These are honey pot fields to catch bots #}
+              <li class="u-off-screen">
+                <label class="website" for="website">Website:</label>
+                <input name="website"
+                       type="text"
+                       class="website"
+                       autocomplete="off"
+                       value=""
+                       id="website"
+                       tabindex="-1" />
+              </li>
+              <li class="u-off-screen">
+                <label class="name" for="name">Name:</label>
+                <input name="name"
+                       type="text"
+                       class="name"
+                       autocomplete="off"
+                       value=""
+                       id="name"
+                       tabindex="-1" />
+              </li>
+              {# End of honey pots #}
+              <li class="p-list__item">
+                <button type="submit" class="p-button--positive">Submit</button>
+              </li>
+            </ul>
+            <div class="u-off-screen">
+              <label for="Comments_from_lead__c">
+                <h3 class="p-heading--4">Your comments</h3>
+                <textarea id="Comments_from_lead__c"
+                          name="Comments_from_lead__c"
+                          rows="5"
+                          maxlength="2000"></textarea>
+              </label>
+            </div>
+            <input type="hidden"
+                   aria-hidden="true"
+                   aria-label="hidden field"
+                   name="formid"
+                   value="5919" />
```

Comment:
do we have this form in Marketo already?

Reply:
Yes this is ready